clientv3/config.go

```diff
@@ -43,6 +43,10 @@ type Config struct {
 
 	// Logger is the logger used by client library.
 	Logger Logger
+
+	// RequireLeader is true, and then it returns rpctypes.ErrNoLeader
+	// when there is no leader when responding to client requests.
+	RequireLeader bool
```

Comment:
this should not be part of the configuration. i think this should be set per request.